packages/core/__tests__/__snapshots__/global.js.snap

```diff
@@ -3,40 +3,40 @@
 exports[`basic 1`] = `
 <head>
   <style
-    data-emotion="css"
+    data-emotion="css-global"
   >
     
-    @-webkit-keyframes animation-ocj8pk{from,to{color:green;}50%{color:hotpink;}}
+    @font-face{font-family:some-name;}
   </style>
   <style
-    data-emotion="css"
+    data-emotion="css-global"
   >
     
-    @keyframes animation-ocj8pk{from,to{color:green;}50%{color:hotpink;}}
+    @import url('something.com/file.css');
   </style>
   <style
     data-emotion="css-global"
   >
     
-    @font-face{font-family:some-name;}
+    html{background-color:hotpink;}
   </style>
   <style
     data-emotion="css-global"
   >
     
-    @import url('something.com/file.css');
+    h1{-webkit-animation:animation-ocj8pk 1s;animation:animation-ocj8pk 1s;}
   </style>
   <style
-    data-emotion="css-global"
+    data-emotion="css"
   >
     
-    html{background-color:hotpink;}
+    @-webkit-keyframes animation-ocj8pk{from,to{color:green;}50%{color:hotpink;}}
   </style>
   <style
-    data-emotion="css-global"
+    data-emotion="css"
   >
     
-    h1{-webkit-animation:animation-ocj8pk 1s;animation:animation-ocj8pk 1s;}
+    @keyframes animation-ocj8pk{from,to{color:green;}50%{color:hotpink;}}
```

Comment:
This reordering of things worries me, could we revert the global refactoring since from what I understand, it's solving a bug or anything, only size/code niceness, right?

Reply:
This change could have fixed a slight bug with `sheet.before` - but I need to give it slightly more thought. Probably better to move this out of this PR so I've reverted this change for now.